src/30_Advanced/Long_list_of_amp-instagram_embeds.html

```diff
@@ -0,0 +1,150 @@
+<!-- 
+  #### Introduction
+
+  A page with a long list of `amp-instagram` embeds.
```

Comment:
what do you think about: "A page with a long list of `amp-instagram` embeds to show how fast and responsive amp pages are, this example contains 15 `amp-instagram` embeds."